HTML/Chess translated (Español — Ajedrez) (MásQuéÉlite).html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Ajedrez — Módulo de Keep Talking and Nobody Explodes</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+    <style>
+        table.chess h4 {
+            margin: 0;
+        }
+        table.chess p {
+            margin: 0;
+        }
+        p, ul {
+            margin: 0;
+        }
+    </style>
+</head>
+    <div class="section">
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Mod de Keep Talking and Nobody Explodes</span>
+                <span class="page-header-section-title">Ajedrez</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/Chess.svg" class="diagram">
+                <h2>En caso de ajedrez</h2>
+                <p class="flavour-text">Bajo presión, el ajedrez se puede sentir como un juego de Hundir la flota.</p>
+                <p>Este módulo se basa en un <b>tablero de ajedrez 6×6</b> <i>(referenciado en la siguiente página)</i> y todas las piezas siguen las normas estándar de movimiento de la FIDE.
```

Comment:
```suggestion
                <p>Este módulo se basa en un <strong>tablero de ajedrez 6×6</strong> <em>(referenciado en la siguiente página)</em> y todas las piezas siguen las normas estándar de movimiento de la FIDE.
```

Reply:
Why it is more recommendable to use 'strong' and 'em' instead of 'b' and 'i'?

Reply:
I'm in a message box lol

Reply:
Not sure tbh but it's recommended in "Red's guide to updating the repo"
https://docs.google.com/document/d/1F1v94J1EpPJqtK4Km_7GJ1mJgH0keWdISoxvhWVB1tI/edit

Reply:
Oh, I didn't find that document. Thank you!